src/main/java/mate/academy/rickandmorty/controller/CharactersController.java

```diff
@@ -0,0 +1,39 @@
+package mate.academy.rickandmorty.controller;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import java.util.List;
+import java.util.Random;
+import lombok.RequiredArgsConstructor;
+import mate.academy.rickandmorty.dto.internal.CharacterDto;
+import mate.academy.rickandmorty.service.CharacterService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@Tag(name = "Character management", description = "Endpoints for managing characters")
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/character")
+public class CharactersController {
+    private static final int CHARACTERS_COUNT = 826;
+
+    private final CharacterService characterService;
+
+    @Operation(summary = "Get character by random id",
+            description = "Returns a character by random id")
+    @GetMapping("/random")
+    public CharacterDto findCharacterByRandomId() {
+        Random random = new Random();

```

Comment:
replace this logic to service
```suggestion
        return characterService.getRandomCharacter();
```